### DISCUSSION TITLE
React integration

### DISCUSSION THREAD

**Discussion by chinespec101** [mdbootstrap/TW-Elements#1517] [Share ideas & request features]
Please integrate TE with React.

**Comment by witmarr97** [comment]
+1 to that

**Comment by smolenski-mikolaj** [comment]
We're working on a dedicated version for React, but for this moment we prepared an easy installation guide for the current version: https://tailwind-elements.com/docs/standard/integrations/react-integration/

**Reply by ukomiljon** [reply]
It does not work with react.
could you make github code integration with react typescript and vise?

**Reply by ukomiljon** [reply]
how can I be a contributor? I would like to help to develop react version of this library.

**Reply by smolenski-mikolaj** [reply]
@ukomiljon it's great that you want to help us. We're going to prepare a dedicated repository, dev branch and contribution guide soon and then we will gladly accept new pull requests. In the meantime, please email us at tailwind@mdbootstrap.com so we can tag you and get back to you when we're ready.

**Reply by ukomiljon** [reply]
I have sent already to the email. Thank you!

**Comment by filipkappa** [comment]
**Current status**

We've published an integration guide for React:
https://tailwind-elements.com/docs/standard/integrations/react-integration/

We are preparing a dedicated React package, you can sign up for the waiting list here:
https://tailwind-elements.com/react/

@ukomiljon Still waiting for your email ;)
We'll be happy to work with any willing contributor :D
